pages/index.js

```diff
@@ -0,0 +1,357 @@
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import styles from "../styles/Home.module.css";
+import Header from "../components/Header";
+import ClientLibraries from "../components/ClientLibraries";
+
+export default function Home() {
+  const [latestPosts, setLatestPosts] = useState();
+
+  const fetchDevToPosts = async () => {
+    const data = await fetch(
+      "https://dev.to/api/articles?username=stripe&state=all"
+    );
+    const posts = await data.json();
+
+    const latest = posts.slice(0, 3);
+
+    setLatestPosts(latest);
+  };
+
+  useEffect(() => {
+    fetchDevToPosts();
+  }, []);
+
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Stripe Developer Resources</title>
+        <link
```

Comment:
I'm getting a console error about this stylesheet - I think it should probably be moved to `_app.js` -

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/102179546/189415297-f798d052-ee72-4be3-b84e-4575a7b464f2.png">

Reply:
I realized we don't even use that font anywhere so i'm removing it.